src/print.c

```diff
@@ -94,11 +94,26 @@ void print_tokens(const Token *tokens)
     printf("\n");
 }
 
-static void print_ast_function_signature(const Signature *sig, int indent)
+static void print_ast_type(const struct AstType *t)
```

Comment:
I keep writing `struct`. Should probably get rid of the typedefs, they also get in the way slightly when doing jump-to-definition.